### ISSUE TITLE
about running the reparameterization.ipynb appear the error

### ISSUE THREAD

**Issue by Luxii44** [WongKinYiu/yolov9#375]
The weight i gained by training the model with "models/detect/yolov9-c.yaml" can't get the new wight with the "reparameterization.ipynb" .i see the "k" is one and the "kr" is two.And the "model.state_dict()[k]" is 64 three-dimensional arrays but the "ckpt['model'].state_dict()[kr]" is 128 three-dimensional arrays.How can I solve the question？
![image](https://github.com/WongKinYiu/yolov9/assets/90958941/f2bac6e4-5410-4327-bf51-eb96737d879e)

**Comment by mizhitian-xiaomi** [comment]
Did you solve it? I also found this error, I use `yolov9-c.yaml`

**Comment by Luxii44** [comment]
> Did you solve it? I also found this error, I use `yolov9-e.yaml`

no,I'm going to try to get the lower dimensions up to the same level as the higher ones.

**Comment by Muyyong** [comment]
I found the same error, I changed the module , but the yaml layers didn't change. Did you change the model structure?

**Comment by tuansunday05** [comment]
Did anyone solve this issue for yolov9-e ? Help me please
